rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -108,7 +108,8 @@ def tick(
 
         elif self._state == State.INIT_EXECUTE_PASS:
             # one tick delay for play role assignment
-            self._state = State.EXECUTE_PASS
+            if len(self.assigned_roles) == 2:
```

Comment:
Nice fix, can you mention this in the description of your PR as a useful thing for future tactic writers?